### DISCUSSION TITLE
How do you trim a video using both "-ss" and "-t" ?

### DISCUSSION THREAD

**Discussion by erickyun** [anasty17/mirror-leech-telegram-bot#1196] [Q&A] [answered]
Hi,

I'm trying to cut a video from the 10 second to the 24 second but when I try
```
/yl opt: postprocessor_args:{"ffmpeg": ["-ss", "10", "-t", "24"]}
```

Only -ss is executed and -t is ignored?
The video starts from the 10 second but doesn't on 24 second?
Both "-ss" "-t" work separately!!
But I can't use them on the same time!

Example:
1. `/yl opt: postprocessor_args:{"ffmpeg": ["-ss", "10"]}`
2. `/yl opt: postprocessor_args:{"ffmpeg": ["-t", "24"]}`

These two command both work without any problem!

But 
3. `/yl opt: postprocessor_args:{"ffmpeg": ["-ss", "10", "-t", "24"]}`

The thirdone doesn't work?? When "-ss" and "-t" are executed together "-t" is ignored? Only "-ss" is executed?

 Am I executing the command wrong could someone help me please?

**Comment by anasty17** [comment, marked as answer]
Bruh this off topic not related to mltb. U can ask yt-dlp or ffmpeg for support.

**Reply by erickyun** [reply]
I have no problem with yt-dlp or ffmpeg!
The Bot ignores the second cmd here?does it work for u?

**Reply by anasty17** [reply]
Bro this postprocessor_args u adding args for postprocessors without using the postprocessors options. This can take effect if add postprocessors option or when i convert to mp3 or to mkv! So the problem that u don't know how those options works with YT-DLP. Read more about YT-DLP options.

**Reply by erickyun** [reply]
I found right cmd👍

```
/yl s opt: force_keyframes_at_cuts:true|external_downloader:{'default': 'ffmpeg'}|external_downloader_args:{'ffmpeg': ['-ss', '30.000', '-to', '40.000', '-vcodec', 'libx264', '-crf', '17', '-aq-mode', '3', '-bf', '16', '-deblock', '-2:-2', '-preset', 'medium', '-pix_fmt', 'yuv420p10', '-c:a', 'aac', '-map', '0']}|final_ext:mkv|writesubtitles:True|subtitleslangs:all,-live_chat|writethumbnail:True|outtmpl:{'pl_thumbnail': ''}|postprocessors: [{'key': 'FFmpegVideoRemuxer', 'preferedformat': 'mkv'}, {'key': 'FFmpegEmbedSubtitle', 'already_have_subtitle': False}, {'already_have_thumbnail': False, 'key': 'EmbedThumbnail'}]
```
